support-frontend/assets/components/nav/nav.tsx

```diff
@@ -0,0 +1,68 @@
+import { css } from '@emotion/react';
+import { between, brand, from } from '@guardian/source-foundations';
+import { Column, Columns } from '@guardian/source-react-components';
+import CountryGroupSwitcher from 'components/countryGroupSwitcher/countryGroupSwitcher';
+import { Container } from 'components/layout/container';
+import type { CountryGroupId } from 'helpers/internationalisation/countryGroup';
+
+const container = css`
+	display: none;
+
+	${from.desktop} {
+		display: block;
+	}
+`;
+
+const divider = css`
+	width: 420px;
+	height: 28px;
+	border-right: 1px solid ${brand[600]};
+	margin-right: 11px;
+
+	${between.leftCol.and.wide} {
+		margin-right: 7px;
+	}
+`;
+
+const columnCssOverrides = css`
+	min-height: 42px;
+	display: flex;
+	align-items: center;
+`;
```

Comment:
```suggestion
const columnCssOverrides = css`
	min-height: 42px;
	display: flex;
	border-left: 1px solid ${brand[600]};
	padding: ${space[2]}px;
`;
```
I think it'll probably be simpler to put the border on the left of the main column and offset it from the country switcher with padding. Also better to use the Source units where the spacing in Figma is only off from them by a pixel or so- it helps us keep things consistent.

Reply:
The "divider" Column has a fixed height so we can have the border that doesnt stretch the full height of the container. I've made some updates but have added a cssOverride to the divider Column